packages/core/src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -343,6 +345,7 @@ export function throwIfCancelled(scope: CodeWhispererConstants.CodeAnalysisScope
 export async function uploadArtifactToS3(
```

Comment:
Can we create a separate function for testGen in testGenHandler.ts?

Reply:
I want to cover these changes as part of Code Refactoring PR.
I added a task for this in milestone doc as there is some good amount of refactoring, needs to be done.